src/components/core/DpUpload/utils/UppyTranslations.js

```diff
@@ -10,11 +12,9 @@ const de = () => {
         browse: '{browse}',
         maxUploadSize: convertSize('GB', window.dplan.settings.maxUploadSize)
       }),
-      failedToUpload: Translator.trans('form.button.upload.failed', {
-        file: '{file}'
-      }),
+      failedToUpload: german.upload.failed,
```

Comment:
I have a question, how to add `file: '{file}'` param here?

Reply:
We don't need to add it, it's replaced by uppy internally.